Directory.Packages.props

```diff
@@ -26,18 +31,18 @@
     <PackageVersion Include="Microsoft.VisualStudio.Internal.MicroBuild.NpmPack"                     Version="2.0.117" />
     <PackageVersion Include="Microsoft.VisualStudio.Internal.MicroBuild.Swix"                        Version="2.0.115" />
     <PackageVersion Include="Microsoft.VisualStudio.Internal.MicroBuild.Vsman"                       Version="2.0.115" />
-    <PackageVersion Include="Microsoft.VisualStudioEng.MicroBuild.Core"                              Version="1.0.0" />
```

Comment:
Do we no longer need this? Does it get brought in transitively?

Reply:
Kind of. A different-but-similar package is brought in transitively. This was a head scratcher. More details in the PR description and comments below if you're interested.